tests/test_registerutils.py

```diff
@@ -152,52 +157,86 @@ def test_is_registration_supported_RHEL_Family():
     assert utils.is_registration_supported(cfg) is False
 
 
-@patch('cloudregister.registerutils.is_new_registration')
-def test_update_rmt_certs_new_reg(new_reg):
-    new_reg.return_value = True
-    res = utils.update_rmt_certs()
-    assert res is None
+def test_has_rmt_in_hosts_has_ipv4():
+    hosts_content = """
+    # simulates hosts file containing the ipv4 we are looking for in the test
 
+    1.1.1.1   smt-foo.susecloud.net  smt-foo
+    """
+    server = MockServer()
+    with mock.patch('builtins.open', mock.mock_open(read_data=hosts_content)):
+        has_entry = utils.has_rmt_in_hosts(server) 
+    
+    assert True == has_entry
+
+
+def test_has_rmt_in_hosts_has_ipv4_6():
+    hosts_content = """
+    # simulates hosts file containing the ipv4 and iv6 we are looking for
+    # in the test
+
+    1.1.1.1   smt-foo.susecloud.net  smt-foo
+    11:22:33:44::::00   smt-foo.susecloud.net  smt-foo
```

Comment:
That ipv6 format is invalid, should be `11:22:33:44:0:0:0:00` or `11:22:33:44::00`